python/image-content-search/src/imageData/helper/insert.py

```diff
@@ -0,0 +1,29 @@
+
+from helper import batch_execute_statement, logger
+
+# this module
+# adds a new image data to the database
+
+def insert_new_image(image_id, labels):
+
+    statement = 'INSERT INTO tags (image_id, label) values (:image_id, :label)'
+    params_sets = []
+
+    for l in labels:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)